config/channel/resources/kafkachannel-crd.yaml

```diff
@@ -48,6 +48,9 @@ spec:
                   type: integer
                   maximum: 32767
                   default: 1
+                retentionMillis:
+                  description: retentionMillis is the topic retention period (in ms) of a Kafka Topic. This is an optional field. By default, the Channel default retention time would be used
+                  type: integer
```

Comment:
Might want to say that the "default" Channel retention is configured in the ConfigMap ?

Reply:
Cleared up the language here.